vllm/utils.py

```diff
@@ -763,9 +764,30 @@ def json_map_leaves(func: Callable[[T], U], value: JSONTree[T]) -> JSONTree[U]:
 
 def flatten_2d_lists(lists: List[List[T]]) -> List[T]:
     """Flatten a list of lists to a single list."""
+    if len(lists) == 1:
+        return lists[0]
     return [item for sublist in lists for item in sublist]
 
 
+def create_torch_tensor_from_1d_list(
+        py_list: List[Any],
+        dtype: torch.dtype,
+        device: Optional[Union[str, torch.device]] = None) -> torch.Tensor:
+    """Convert a list to a torch tensor."""
+    if dtype == torch.int:
+        array_dtype = "i"
+    elif dtype == torch.float:
+        array_dtype = "f"
+    elif dtype == torch.long:
```

Comment:
also add some type alias support for `torch.int32` and `torch.int64` `torch.float32`